CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 
 ### Breaking changes
 
-* Lorem ipsum.
+* New replication instruction: instr_AddRowWithKey
```

Comment:
As we discussed in the core meeting, I still think that this is a `instr_AddRowWithValue` - not _key_. From core's point of view, this is simply an integer value we are setting on a integer column. If / when we introduce unique IDs, we will have a key type (different from `int64_t`) and at that point we can add the "real" `instr_AddRowWithKey`.